src/use/use.py

```diff
@@ -682,7 +648,7 @@ def isfunction(x: Any) -> bool:
 
 
 def ismethod(x: Any) -> bool:
-    return inspect.isfunction(x)
+    return inspect.ismethod(x)
```

Comment:
let's remove ismethod, isfunction. if we import them directly, they end up as globals in use anyway..

Reply:
ok, I agree